sync-team/src/github/tests/mod.rs

```diff
@@ -19,7 +19,11 @@ fn team_create() {
     let user = model.create_user("mark");
     let user2 = model.create_user("jan");
     let gh = model.gh_model();
-    model.create_team(TeamData::new("admins").gh_team(DEFAULT_ORG, "admins-gh", &[user, user2]));
+    model.create_team(TeamData::new_builder("admins").gh_team(
```

Comment:
I had to change `new` because that method wasn't returning Self. So I called the method `new_builder`

Reply:
`build`? It's annoying though, I would just silence the lint on this method.

Reply:
I also agree to silence that lint.

Reply:
Personally, I'd stick with the convention here and rename the method to something like `builder`. We're probably all using code completion, so there's imo no cost in adding a few characters to the method name. 😅 

But not a blocking concern, happy to go with whatever the majority thinks...